components/bl-gallery-component/src/index.js

```diff
@@ -0,0 +1,88 @@
+import { useMemo } from 'react';
+
+import { Gallery } from './components/gallery.js';
+
+const DefaultOptions = {
+  IMAGE_LABEL: 'Image',
+  SEPARATOR: 'of',
+  FADE_DURATION: 600,
+  IMAGE_FADE_DURATION: 600,
+  POSITION_FROM_TOP: 50,
+  RESIZE_DURATION: 700,
+};
+
+export default function GalleryComponent({ component }) {
+  const {
+    imagesData,
+    alwaysShowNavOnTouchDevices,
+    imageLabel,
+    separator,
+    disableScrolling,
+    fadeDuration,
+    imageFadeDuration,
+    positionFromTop,
+    resizeDuration,
+    showImageCount,
+    wrapAround,
+  } = component;
+
+  const options = useMemo(() => {
+    const {
+      IMAGE_LABEL,
+      SEPARATOR,
+      FADE_DURATION,
+      IMAGE_FADE_DURATION,
+      POSITION_FROM_TOP,
+      RESIZE_DURATION,
+    } = DefaultOptions;
+
+    return {
+      alwaysShowNavOnTouchDevices: alwaysShowNavOnTouchDevices === 'true',
+      imageLabel: imageLabel || IMAGE_LABEL,
```

Comment:
imageLabel: imageLabel || DefaultOptions.IMAGE_LABEL,

or remove namespace `DefaultOptions` and use `const DEFAULT_IMAGE_LABEL = ...`